python/pysmurf/client/util/smurf_util.py

```diff
@@ -878,8 +916,42 @@ def read_stream_data(self, datafile, channel=None,
         if array_size is not None:
             phase.resize(array_size, phase.shape[1])
 
-        return t, phase, mask
+        if return_header:
+            return t, phase, mask, header_dict
+        
+        else:
+            return t, phase, mask
+
 
+    def header_to_tes_bias(self, header, n_tes_bias=15):
```

Comment:
Need description of n_tes_bias in docstring.

Reply:
Pushing update with doc string